client/my-sites/plugins/plugin-management-v2/plugin-action-status/index.tsx

```diff
@@ -24,7 +24,7 @@ export default function PluginActionStatus( {
 }: Props ) {
 	// Group statuses by status type(completed, error, inProgress)
 	const groupedStatues = currentSiteStatuses.reduce( ( group, plugin ) => {
-		const { status } = plugin;
+		const status = plugin.actualStatus || plugin.status;
```

Comment:
What's the difference between these two fields?

Reply:
I have renamed it to `displayStatus`. Basically this is used only to show the status on the UI. Plugins are filtered based on the `status`.